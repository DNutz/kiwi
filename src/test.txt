férié,
à noter que,
sacré(e),
étaler,
la fermeture annuelle,
le trekking,
le patrimoine,
une region,
le tourisme ludique,
une recette,
le dépaysement ,
L’album photo,
Gnome de jardin,
Pronier de portre,
Reconnection,
Individuel,
Spontané,
Sûr ,
se reposer,
se détendre,
élargir ses connaissance,
tolérance,
S’agrandir,
affaire,
organisés - tour organisé,
aventure - routière,
lune de miel,
croisière,
sportif,
religieux,
d’étude,
- romantique,
- de comédie,
- d’action,
- biographique,
- d’horreur,
- de guerre,
- documentaire,
- musical,
- de science-fiction,
- de western,
- thriller,
- fantastique,
- de propagande ,
Le petit écran,
La grand écran,
Le 7ème art ,
Une série télévisé,
Les télé-réalités,
Le journal télévisé,
Une émission = programe télévisé,
Le dessins animés,
La météo ,
Un site de rencontre,
Faire des rencontres,
Les données,
Un trait de caractère,
Un défaut,
Un feuilleton,
,
