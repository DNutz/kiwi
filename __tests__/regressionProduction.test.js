import { setup, teardown, getFile } from '../testUtils/test.utils'

let chromeless = null

beforeAll(() => { chromeless = setup() })
afterAll(async () => { await teardown(chromeless) })

test('+++ home renders correctly', async () => {
  const html = await chromeless
    // .goto(global.config.baseUrl)
    .goto("https://kiwi-prod.firebaseapp.com")
    .wait('div#root')
    .evaluate(() => document.querySelector('div#root').innerHTML)
  expect(html).toMatchSnapshot()
})

test('+++ home renders correctly (visual)', async () => {
  const screenshotPath = await chromeless
  // .goto(global.config.baseUrl)
    .goto("https://kiwi-prod.firebaseapp.com").wait('div#root').screenshot()
  const screenshot = await getFile(screenshotPath)
  expect(screenshot).toMatchImageSnapshot()
})