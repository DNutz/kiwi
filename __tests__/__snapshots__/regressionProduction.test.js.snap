// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`+++ home renders correctly 1`] = `"<h1 style=\\"font-family: sans-serif;\\">Component: PeterTest</h1><p style=\\"color: blue; font-family: sans-serif;\\">Look Peter the component works!</p><p style=\\"color: blue; font-family: sans-serif;\\">Look Peter the component works!</p><p style=\\"color: blue; font-family: sans-serif;\\">Look Peter the component works!</p><p style=\\"color: blue; font-family: sans-serif;\\">Look Peter the component works!</p><p style=\\"color: blue; font-family: sans-serif;\\">Look Peter the component works!</p>"`;
